InfiniteRecharge/src/main/java/frc/robot/subsystem/BitBucketSubsystem.java

```diff
@@ -110,5 +113,7 @@ public void initialize() {
 
     public abstract void disable();
 
-    public abstract BaseTalon[] getTalons();
+    protected List<BaseTalon> talons = new ArrayList<BaseTalon>();
+    protected abstract void listTalons();
```

Comment:
listTalons sounds like a function to print a bunch of talons to the console. Can you rename this addTalonsToSubsystemList? or addTalonsToList()